Snakemake/workflow/rules/prepare_files_for_tsinfer.smk

```diff
@@ -1,115 +1,116 @@
+if config['ancestral_allele'] is not None:
+    ancestral_file = config['ancestral_allele']
+else:
+    ancestral_file = rules.combine_pos_ancestral.output	
+
```

Comment:
Here, we figure out which is our ancestral file - but then we run the same steps regardles